dialogs.py

```diff
@@ -237,18 +246,18 @@ def __init__(self, dir, gui, parent=None):
 
                 pidlabel = QLabel(self)
                 pidlabel.setTextFormat(QtCore.Qt.RichText)
-                pidlabel.setText(pre + l[1] + post)
+                pidlabel.setText(pre + ll[1] + post)
                 self.gridLayout.addWidget(pidlabel, i, 1, 1, 1)
 
-                l = open(dir + file).readlines()
+                lines = open(dir + file).readlines()
```

Comment:
This isn't really the functional change part of this PR but I notice this file never gets closed